File: isma-compiler/hsm-core/src/main/java/ru/nstu/isma/compiler/hsm/core/var/HMEquation.java
package ru.nstu.isma.compiler.hsm.core.var;

import ru.nstu.isma.compiler.hsm.core.exp.HMExpression;

import java.io.Serializable;

/**
 * Created by Bessonov Alex
 * Date: 24.10.13
 * Time: 23:29
 */
public class HMEquation extends HMVariable implements Serializable {

    protected HMExpression rightPart;

    public HMExpression getRightPart() {
        return rightPart;
    }

    public void setRightPart(HMExpression rightPart) {
        this.rightPart = rightPart;
    }

}

File: isma-next-hsm/src/main/kotlin/ru/nstu/isma/hsm/var/HMEquation.kt
package ru.nstu.isma.hsm.`var`

import ru.nstu.isma.hsm.exp.HMExpression
import java.io.Serializable

/**
 * Created by Bessonov Alex
 * Date: 24.10.13
 * Time: 23:29
 */
open class HMEquation : HMVariable(), Serializable {
    open var rightPart: HMExpression? = null
}